CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Removed
+
+- Support for discontinued non-managed object storage product
```

Comment:
This was documented as non-breaking change in the API (no version bumps) so we could follow that decision in tooling as well. 🤔 Maybe we could say here that this is treated as non-breaking change as all users have been migrated to the new managed object storage product.